src/components/AppNavigation/AppNavigation.vue

```diff
@@ -110,7 +110,7 @@ export default {
 		toggleNavigation(state) {
 			this.open = state || !this.open
 			const bodyStyles = getComputedStyle(document.body)
-			const animationLength = parseInt(bodyStyles.getPropertyValue('--animation-quick')) | 100
+			const animationLength = 1.5 * (parseInt(bodyStyles.getPropertyValue('--animation-quick')) | 100)
```

Comment:
Maybe but the multiplier on line 119 instead?
And a nice comment explaining? :p

Reply:
Ok, will do that.

Reply:
Fixed.